src/software/simulation/er_force_simulator.cpp

```diff
@@ -111,18 +73,110 @@ void ErForceSimulator::setBallState(const BallState& ball_state)
     *(command_simulator->mutable_ssl_control())     = *simulator_control;
     *(simulator_setup_command->mutable_simulator()) = *command_simulator;
 
-
     er_force_sim->handleSimulatorSetupCommand(simulator_setup_command);
 }
 
 void ErForceSimulator::addYellowRobots(const std::vector<RobotStateWithId>& robots)
 {
-    // TODO (#2283): add robots
+    addRobots(robots, gameController::Team::YELLOW);
 }
 
 void ErForceSimulator::addBlueRobots(const std::vector<RobotStateWithId>& robots)
 {
-    // TODO (#2283): add robots
+    addRobots(robots, gameController::Team::BLUE);
+}
+
+void ErForceSimulator::addRobots(const std::vector<RobotStateWithId>& robots,
+                                 gameController::Team side)
+{
+    auto simulator_setup_command = std::make_unique<amun::Command>();
+    simulator_setup_command->mutable_simulator()->set_enable(true);
+
+    robot::Specs ERForce;
+    robotSetDefault(&ERForce);
+
+    // Initialize Team Robots at the bottom of the field
+    if (side == gameController::Team::BLUE)
```

Comment:
Is it possible to extract the common parts in the conditions (forloop and handleSimulatorSetupCommand)?